### PR TITLE
Add sorting to document list view

### PR DESCRIPTION
**Pull request by eduardoboucas**

### CODE REVIEW

**Comment by eduardoboucas** [comment]
This closes #48.

File: dadi/frontend/views/DocumentList/DocumentList.jsx [hunk 1]
```diff
@@ -103,11 +118,22 @@ class DocumentList extends Component {
...
+    const {
+      actions,
+      collection,
+      filter,
+      order,
+      page,
+      sort,
+      state
+    } = this.props
+    const urlSearch = state.routing.locationBeforeTransitions.search
```
Comment (mingard) [inline dadi/frontend/views/DocumentList/DocumentList.jsx]: @eduardoboucas remove this?

Comment (eduardoboucas) [inline dadi/frontend/views/DocumentList/DocumentList.jsx]: Totally!

Comment (mingard) [inline dadi/frontend/views/DocumentList/DocumentList.jsx]: Done

**Pull request merged**